### PR TITLE
GA known issues for the SDK and updates to resolved items

### PR DESCRIPTION
**Pull request by marcpopMSFT**

### CODE REVIEW

File: release-notes/5.0/5.0-known-issues.md [hunk 1]
```diff
@@ -175,15 +178,33 @@ The Virtualization component is new in RC1 and there are some fixes we didn't fi
...
+   **Workaround**
+
+   Add a *Directory.Build.Targets* file with the following content
+
+   ```xml
+   <Project>
+     <Target Name="EnsureNETCoreAppRuntime" />
+   </Project>
+   ```
+2. Starting with .NET 5 preview 8, the .NET SDK requires Visual Studio 16.8 or newer. The SDK will not load in 16.7 and if you try to target `net5.0`, you will get compile errors. The error will be improved in a future release of 16.7 to advise you to update your Visual Studio installation. https://github.com/dotnet/msbuild/pull/5826
```
Comment (marcpopMSFT) [inline release-notes/5.0/5.0-known-issues.md:207]: @rbhanda We've had a bunch of confusion about .net 5 requiring 16.8.  I'm not sure the known issues list is the right place for this (it's listed in our installer but people don't notice that).  Is there a better spot for that to be documented?

Comment (rbhanda) [reply release-notes/5.0/5.0-known-issues.md:207]: We will have release notes for 5.0 so an equivalent of 5.0 release notes like https://github.com/dotnet/core/blob/master/release-notes/5.0/preview/5.0.0-rc.2.md#visual-studio-compatibility seems like the best place for this documentation.

**Comment by rbhanda** [comment]
These are added to https://github.com/dotnet/core-private/pull/81/files

**Comment by marcpopMSFT** [comment]
Tracked in a central PR.

**Pull request closed without merging**